config/jobs/kubernetes/sig-arch/conformance-audit.yaml

```diff
@@ -1,3 +1,54 @@
+presubmits:
+  kubernetes/kubernetes:
+    # same as ci-audit-kind-conformance, but a presubmit job
+    - name: pull-audit-kind-conformance
```

Comment:
nit: we usually name presubmits `pull-$repo-$description`

Reply:
We should also consider replacing the ga-only-parallel presubmit with just a ga-only presubmit that triggers on conformance changes and runs conformance normally. It takes 2h but vetting conformance is important.

cc @liggitt

Reply:
@liggitt this is not aimed to run on all PRs.

Reply:
> @liggitt this is not aimed to run on all PRs.

sure, I was just responding to ben's comment

Reply:
> Right now we only run the non-serial version

And that explicitly excludes serial tests? Because of runtime concerns? It's not necessary technically, the same job can run the parallel tests in parallel and the serial ones sequentially, ginkgo v2 handles that.